web/server/codechecker_server/database/run_db_model.py

```diff
@@ -360,12 +360,16 @@ class Comment(Base):
     bug_hash = Column(String, nullable=False, index=True)
     author = Column(String, nullable=False)
     message = Column(Binary, nullable=False)
+    kind = Column(Integer,
+                  nullable=False,
+                  server_default="0")
```

Comment:
Maybe a comment about the default value here would be nice. '0' means a comment type by the user tight?